tsl/src/continuous_aggs/job.c

```diff
@@ -75,12 +75,23 @@ ts_continuous_agg_job_add(int32 raw_table_id, int64 bucket_width, Interval *refr
 		refresh_interval =
 			continuous_agg_job_get_default_schedule_interval(raw_table_id, bucket_width);
 
+	namestrcpy(&proc_name, "");
+	namestrcpy(&proc_schema, "");
+	namestrcpy(&owner, "");
```

Comment:
Not sure why these are empty. The table definition says `NOT NULL` for all columns, so what does it mean to have an empty `proc_name` (for example)?